### PR TITLE
RHDEVDOCS-2072: Doc; [ODC] UI enhancements based on customer feedback

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; [ODC] UI enhancements based on customer feedback
Doc JIRA: https://issues.redhat.com/browse/RHDEVDOCS-2072
Dev JIRAs: https://issues.redhat.com/browse/ODC-3418
Fix Version: Openshift 4.14 Freeze

Doc Preview: https://61214--docspreview.netlify.app/openshift-enterprise/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#odc-deploying-container-image_odc-creating-applications-using-developer-perspective

SME Review: @vikram-raj 
QE Review: @sanketpathak 
Peer Review: @abrennan89

### CODE REVIEW

File: modules/odc-deploying-container-image.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console and are in the *Developer* perspective.
+
+.Procedure
+
+. In the *+Add* view, click the *Container images* link to view the *Deploy Images* page.
+. In the *Image* section:
+.. Select *Image name from external registry*, to deploy an image from a private registry.
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/odc-deploying-container-image.adoc:19]: Made this change! Thanks

File: modules/odc-deploying-container-image.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
+.. Select an icon for your image in the *Runtime icon* tab.
+. In the *General* section:
+.. In the *Application name* field, enter a unique name for the application grouping.
+.. In the *Name* field, enter a unique name to identify the resources created for this component.
+. In the *Resources* section, select the resource type to generate:
+.. Select *Deployment* to enable declarative updates for Pods and ReplicaSets.
+.. Select *Deployment Config* to define the template for a pod and manage deploying new images and configuration sources.
+.. Select *Knative Service* to enable scaling to zero when idle.
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/odc-deploying-container-image.adoc:28]: Made this change! Thanks

File: modules/odc-deploying-container-image.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+. In the *Image* section:
+.. Select *Image name from external registry*, to deploy an image from a private registry.
+.. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
+.. Select an icon for your image in the *Runtime icon* tab.
+. In the *General* section:
+.. In the *Application name* field, enter a unique name for the application grouping.
+.. In the *Name* field, enter a unique name to identify the resources created for this component.
+. In the *Resources* section, select the resource type to generate:
+.. Select *Deployment* to enable declarative updates for Pods and ReplicaSets.
+.. Select *Deployment Config* to define the template for a pod and manage deploying new images and configuration sources.
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/odc-deploying-container-image.adoc:27]: Made this change! Thanks

File: modules/odc-deploying-container-image.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+.Prerequisites
+
+* You have logged in to the {product-title} web console and are in the *Developer* perspective.
+
+.Procedure
+
+. In the *+Add* view, click the *Container images* link to view the *Deploy Images* page.
+. In the *Image* section:
+.. Select *Image name from external registry*, to deploy an image from a private registry.
+.. Select *Image stream tag from internal registry*, to deploy an image from a public or a private registry.
```
Comment (vikram-raj) [inline modules/odc-deploying-container-image.adoc:20]:
```suggestion
.. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
```

File: modules/odc-deploying-container-image.adoc (hunk 1 above)
Comment (vikram-raj) [inline modules/odc-deploying-container-image.adoc:19]:
```suggestion
.. Select *Image name from external registry*, to deploy an image from a public or a private registry.
```

File: modules/odc-deploying-container-image.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+. In the *+Add* view, click the *Container images* link to view the *Deploy Images* page.
+. In the *Image* section:
+.. Select *Image name from external registry*, to deploy an image from a public or a private registry.
+.. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
+.. Select an icon for your image in the *Runtime icon* tab.
+. In the *General* section:
+.. In the *Application name* field, enter a unique name for the application grouping.
+.. In the *Name* field, enter a unique name to identify the resources created for this component.
+. In the *Resources* section, select the resource type to generate:
```
Comment (vikram-raj) [inline modules/odc-deploying-container-image.adoc:25]:
```suggestion
. In the *Resource type* section, select the resource type to generate:
```

**Review by vikram-raj — COMMENTED** [review]
LGTM

**Comment by sanketpathak** [comment]
LGTM

File: modules/odc-deploying-container-image.adoc [hunk 6]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="odc-deploying-container-image_{context}"]
+= Creating applications by deploying container image
+
+You can deploy existing images from an image registry or image stream tag to deploy it on to the {product-title}.
```
Comment (abrennan89) [inline modules/odc-deploying-container-image.adoc:9]:
```suggestion
You can use an external image registry or an image stream tag from an internal registry to deploy an image on your cluster.
```
suggestion for readability

File: modules/odc-deploying-container-image.adoc [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+.. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
+.. Select an icon for your image in the *Runtime icon* tab.
+. In the *General* section:
+.. In the *Application name* field, enter a unique name for the application grouping.
+.. In the *Name* field, enter a unique name to identify the resources created for this component.
+. In the *Resource type* section, select the resource type to generate:
+.. Select *Deployment* to enable declarative updates for Pods and ReplicaSets.
+.. Select *DeploymentConfig* to define the template for a pod and manage deploying new images and configuration sources.
+.. Select *Serverless Deployment* to enable scaling to zero when idle.
+. Click *Create* to create the application. You can see the build status of the application in the *Topology* view.
```
Comment (abrennan89) [inline modules/odc-deploying-container-image.adoc:29]:
```suggestion
. In the *+Add* view, click *Container images* to view the *Deploy Images* page.
. In the *Image* section:
.. Select *Image name from external registry* to deploy an image from a public or a private registry, or select *Image stream tag from internal registry* to deploy an image from an internal registry.
.. Select an icon for your image in the *Runtime icon* tab.
. In the *General* section:
.. In the *Application name* field, enter a unique name for the application grouping.
.. In the *Name* field, enter a unique name to identify the resources created for this component.
. In the *Resource type* section, select the resource type to generate:
.. Select *Deployment* to enable declarative updates for `Pod` and `ReplicaSet` objects.
.. Select *DeploymentConfig* to define the template for a `Pod` object, and manage deploying new images and configuration sources.
.. Select *Serverless Deployment* to enable scaling to zero when idle.
. Click *Create*. You can see the build status of the application in the *Topology* view.
```

**Review by abrennan89 — COMMENTED** [review]
Left some comments, thanks @shreyasiddhartha !

File: modules/odc-deploying-container-image.adoc (hunk 6 above)
Comment (shreyasiddhartha) [reply modules/odc-deploying-container-image.adoc:9]: Made this change! Thanks.

File: modules/odc-deploying-container-image.adoc (hunk 7 above)
Comment (shreyasiddhartha) [reply modules/odc-deploying-container-image.adoc:29]: Made this change. Thanks!

File: modules/odc-deploying-container-image.adoc [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+.. Select *Image name from external registry* to deploy an image from a public or a private registry, or select *Image stream tag from internal registry* to deploy an image from an internal registry.
+.. Select an icon for your image in the *Runtime icon* tab.
+. In the *General* section:
+.. In the *Application name* field, enter a unique name for the application grouping.
+.. In the *Name* field, enter a unique name to identify the resources created for this component.
+. In the *Resource type* section, select the resource type to generate:
+.. Select *Deployment* to enable declarative updates for `Pod` and `ReplicaSet` objects.
+.. Select *DeploymentConfig* to define the template for a `Pod` object, and manage deploying new images and configuration sources.
+.. Select *Serverless Deployment* to enable scaling to zero when idle.
+. Click *Create*. You can see the build status of the application in the *Topology* view.
```
Comment (abrennan89) [inline modules/odc-deploying-container-image.adoc:28]:
```suggestion
. Click *Create*. You can view the build status of the application in the *Topology* view.
```
minor edit 🙂

**Comment by abrennan89** [comment]
/lgtm
Thanks @shreyasiddhartha 🙂

File: modules/odc-deploying-container-image.adoc (hunk 8 above)
Comment (shreyasiddhartha) [reply modules/odc-deploying-container-image.adoc:28]: Made this change. Thanks!

**Pull request merged**